raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml

```diff
@@ -55,11 +55,11 @@ scenario:
             - wait_blocks: 1
             # Assert that correct amount was tranferred
             - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 999_000_000_000_000_000}
-            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_001_000_000_000_000_000}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000}
```

Comment:
You need to set `total_deposit: 1_000_000_000_000_000_000` to `total_deposit: 0`, right?

Reply:
Good catch 👀